testing/e2e/bazel_run_docker.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+set -ex
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+source ./testing/e2e/util.sh
+
+# Must be invoked from the root of the repo.
+ROOT=$PWD
+
+function test_bazel_run_docker_build_clean() {
+  cd "${ROOT}"
+  for target in $(eval $CONTAINER_IMAGE_TARGETS_QUERY);
```

Comment:
Comment explaining what "CONTAINER_IMAGE_TARGETS_QUERY" is supposed to have at the top of this file and where it's set. I couldn't find where it's set by looking at this PR.

Reply:
fixed